hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteCommitCallbackConfig.java

```diff
@@ -91,13 +103,44 @@ public Builder withCallbackHttpApiKey(String apiKey) {
       return this;
     }
 
+    public Builder withCallbackKafkaBootstrapServers(String kafkaBootstrapServers) {
```

Comment:
I am thinking one thing: the config options of the callback feature for Kafka are left in `hudi-client`, while the implementation hosts in `hudi-utilities`, if it's suitable.

Reply:
> 
> 
> I am thinking one thing: the config options of the callback feature for Kafka are left in `hudi-client`, while the implementation hosts in `hudi-utilities`, if it's suitable.

It is truly a problem. since kafka callback implementation is placed in hudi-utilities, the config should be there too. will move it